### PR TITLE
[Opset13][pyAPI] Python API opset13 init and NMSRotated-13

### PR DESCRIPTION
**Pull request by mitruska**
### Details:
 - Initialization of Opset13 python API
 - Add python API for NMSRotated-13

### Tickets:
 - 115300

### CODE REVIEW

File: src/bindings/python/src/openvino/runtime/opset13/ops.py [hunk 1]
```diff
@@ -0,0 +1,59 @@
+# -*- coding: utf-8 -*-
+# Copyright (C) 2018-2023 Intel Corporation
+# SPDX-License-Identifier: Apache-2.0
+
+"""Factory functions for all ngraph ops."""
```
Comment (mmikolajcz) [inline src/bindings/python/src/openvino/runtime/opset13/ops.py:5]:
```suggestion
"""Factory functions for all openvino ops."""
```
Shouldn't we switch from ngraph to ov?

File: src/bindings/python/src/openvino/runtime/opset13/ops.py [hunk 2]
```diff
@@ -0,0 +1,59 @@
...
+    score_threshold: NodeInput,
+    sort_result_descending: Optional[bool] = True,
+    output_type: Optional[str] = "i64",
+    clockwise: Optional[bool] = True,
+    name: Optional[str] = None,
+) -> Node:
+    """Return a node which performs NMSRotated.
+
+    :param boxes: Tensor with box coordinates [x_ctr, y_ctr, width, height, angle_radians].
+    :param scores: Tensor with box scores.
```
Comment (mmikolajcz) [inline src/bindings/python/src/openvino/runtime/opset13/ops.py:39]: Maybe add information about expected shapes and types like in reference?

**Review by p-wysocki — APPROVED** [review]
LGTM, after @mmikolajcz's comments. Also, when introducing opset12 at https://github.com/openvinotoolkit/openvino/pull/18481 there was a decision to bump the opset in imports of all test files, should we keep doing that?

File: src/bindings/python/src/openvino/runtime/opset13/ops.py [hunk 3]
```diff
@@ -0,0 +1,59 @@
...
+
+
+@nameable_op
+def nms_rotated(
+    boxes: NodeInput,
+    scores: NodeInput,
+    max_output_boxes_per_class: NodeInput,
+    iou_threshold: NodeInput,
+    score_threshold: NodeInput,
+    sort_result_descending: Optional[bool] = True,
```
Comment (jiwaszki) [inline src/bindings/python/src/openvino/runtime/opset13/ops.py:31]:
```suggestion
    sort_result_descending: bool = True,
```

**Review by jiwaszki — COMMENTED** [review]
Just follow-up with update of opset in tests.

File: src/bindings/python/src/openvino/runtime/opset13/ops.py (hunk 3 above)
Comment (mitruska) [reply src/bindings/python/src/openvino/runtime/opset13/ops.py]: Done

File: src/bindings/python/src/openvino/runtime/opset13/ops.py (hunk 2 above)
Comment (mitruska) [reply src/bindings/python/src/openvino/runtime/opset13/ops.py]: Done

File: src/bindings/python/src/openvino/runtime/opset13/ops.py (hunk 1 above)
Comment (mitruska) [reply src/bindings/python/src/openvino/runtime/opset13/ops.py]: Aligned with the second suggestion

**Comment by mitruska** [comment]
> LGTM, after @mmikolajcz's comments. Also, when introducing opset12 at #18481 there was a decision to bump the opset in imports of all test files, should we keep doing that?

The opset imports in the tests have been updated (as discussed, redesign of this approach will be considered by @jiwaszki)

**Review by mmikolajcz — APPROVED** [review]

**Review by jiwaszki — APPROVED** [review]

**Review by p-wysocki — APPROVED** [review]

**Pull request merged**